tex/multinomial_tex/nips_multinomial_2018.tex

```diff
@@ -129,43 +130,86 @@ \subsection{Data}
 
 \subsection{Decision Tree Surrogate} \label{sec:surrogate_dt}
 
-Given a learned function $g$, a set of learned output responses $g(\mathbf{X}) = \mathbf{\hat{Y}}$, and a tree splitting and pruning approach $\mathcal{A}$, a global -- or over all $\mathbf{X}$ -- surrogate decision tree $h_{\text{tree}}$ can be extracted such that $h_{\text{tree}}(\mathbf{X}) \approx g(\mathbf{X})$:
+Given a learned function $g$, a set of learned output responses $g(\mathbf{X}) = \mathbf{\hat{Y}}$, and a tree splitting and pruning(pre or post pruning) approach $\mathcal{A}$, a surrogate decision tree $h_{\text{tree}}$ can be constructed to extract comprehensible and faithful concepts such that $h_{\text{tree}}(\mathbf{X}) \approx g(\mathbf{X})$:
 
 \begin{equation}
 \mathbf{X}, g(\mathbf{X}) \xrightarrow{\mathcal{A}} h_{\text{tree}}
 \end{equation}
 
 \begin{figure}[htb]
-	\begin{center}
-		\includegraphics[height=200pt]{img/simple_tree_prepruned_simulated.png}
-		\caption{}
-		\label{fig:surrogate_dt}
-	\end{center}
+	\begin{subfigure}[tb]{.5\textwidth}
+		\begin{center}
+			\includegraphics[height=150pt, width=\linewidth]{img/simple_tree_prepruned_simulated.png}
+			\caption{Explain the figure}
+			\label{fig:surrogate_dt}
+		\end{center}
+	\end{subfigure}%
+	\begin{subfigure}[tb]{.5\textwidth}
+		\begin{center}
+			\includegraphics[height=70pt, width=\linewidth]{img/decision_stumps_txt.png}
+			\caption{Explain the figure}
+			\label{fig:surrogate_dt}
+		\end{center}
+	\end{subfigure}%
+	\caption{}
 \end{figure}
 
-Prescribed methods for training $h_{\text{tree}}$ do exist \cite{dt_surrogate1} \cite{dt_surrogate2}. In practice, straightforward cross-validation and pruning approaches are often sufficient. Moreover, comparing cross-validated training error to traditional training error can give an indication of the stability of the single decision tree $h_{\text{tree}}$.
-
-Elegantly handles high cardinality targets.
+In this learning task, the target labels are derived from the base estimators learned function. \cite{dt_surrogate1} \cite{dt_surrogate2} provide detained information for training $h_{\text{tree}}$. In practice, straightforward cross-validation and pruning approaches are often sufficient. Moreover, comparing cross-validated training error to traditional training error can give an indication of the stability of the single decision tree $h_{\text{tree}}$. Such form of surrogate models can elegantly handle high cardinality targets maintaining high level of fidelity and faithfulness to the Oracle(base estimator function $g$).
```

Comment:
Still unsure about the term `base estimator`. What does it mean in this context? I think its better to refer to `$g$` as it is. Also, `base estimator` makes me think we are doing some type of ensembling as this term is used a lot in that literature. 

Also, not sure about using the term `Oracle` here. Better to leave the sentence as:
`Such form of surrogate models can elegantly handle high cardinality targets maintaining high level of fidelity and faithfulness to the function $g$`